src/notifications/notifications.ts

```diff
@@ -0,0 +1,84 @@
+export const NEW_NOTIFS = [
```

Comment:
Even if there's nothing yet to add, I would set up the empty `OLD_NOTIFS` array now, just so it's ready and reduces the chance for mistakes.

Reply:
Here the iteration throught each notification is easy, and otherwise if we want to make then we have to make `indices` array. Again it will become comple.

Reply:
If you import like you are doing in `notifications/background/index` module (`import * as notifications from './notifications'`), they're going to be easy to iterate over even if they are separate exports. 
Anyway, this is not really important. However, one important thing I did just think of, which would reduce errors when defining these, would be to come up with some `NotifDefinition` type (name is just a suggestion). Then it will be very obvious if we accidentally add some new notif with a field typo or missing fields - build and CI should error